macros/create_latest_version_view.sql

```diff
@@ -15,7 +15,11 @@
         {% set create_view_sql -%}
             -- this syntax may vary by data platform
             CREATE OR REPLACE VIEW {{ new_relation }}
-            AS SELECT * FROM {{ this }}
+            AS SELECT * FROM {{ this }};
+              ALTER TABLE {{ this }} 
+              SET OPTIONS (
+                description = "{{ model.get('description', '') }}" || "\n\nhttps://github.com/mozilla/mozdbt/blob/main/models/" || "{{ model.path }}"
+              )
```

Comment:
This might have to be an `ALTER VIEW`. Alternatively we could do this in one statement:
```suggestion
            OPTIONS (
                description="{{ model.get('description', '') }}" || "\n\nhttps://github.com/mozilla/mozdbt/blob/main/models/" || "{{ model.path }}"
            )
            AS SELECT * FROM {{ this }}
```